### PR TITLE
translate: allow setting format of text

### PR DESCRIPTION
**Pull request by stephenplusplus**
_Continuing #1574_

See #1574 for the discussion of this feature.

@innovative1 thanks for the idea and the original implementation! Can you take a look at this one?

### CODE REVIEW

File: packages/translate/src/index.js [hunk 1]
```diff
@@ -268,6 +269,7 @@ Translate.prototype.getLanguages = function(target, callback) {
  *     target ISO 639-1 language code to translate the source input to. (e.g.
  *     `en` for English). If an object, you may also specify the source
  *     language.
+ * @param {string} options.format - Set the text's format as `html` or `text`.
```
Comment (callmehiphop) [inline packages/translate/src/index.js]: We should probably mark this as optional and mention that the default is text.

**Comment by callmehiphop** [comment]
@stephenplusplus can we add a system test for this?

**Comment by stephenplusplus** [comment]
PTAL!

File: packages/translate/src/index.js (hunk 1 above)
Comment (stephenplusplus) [inline packages/translate/src/index.js]: `options` itself is marked as optional. I elaborated on what happens if a user doesn't provide a format.

**Pull request merged**